src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -1064,20 +1109,34 @@ function getCategories(
     return [];
 }
 
-function getStackingConfig(stackByAttribute: any, options: any) {
-    const stackingValue = 'normal';
-    const { type, stacking } = options;
+function getCategoriesForDualAxis(
```

Comment:
Is this unit tested? I didn't see a specific test in chartOptionsBuilder.spec.ts

Reply:
I will add unit test soon :)